File: 2.2 Software Technologies/04.1 JAVA-SYNTAX - EXERCISE/13.Reverse String/src/ReverseString.java
import java.util.Scanner;

public class ReverseString {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String input = scanner.nextLine();

        String reverse = new StringBuilder(input).reverse().toString();

        System.out.println(reverse);
    }
}

File: 3.1.1 C# Advanced/05.1 EXERCISE-MANUAL STRING PROCESSING/01.ReverseString/ReverseString.cs
using System;
using System.Text;

namespace _01.ReverseString
{
    public class ReverseString
    {
        public static void Main()
        {
            var text = Console.ReadLine();

            var sb = new StringBuilder();

            for (int i = text.Length - 1; i >= 0; i--)
            {
                sb.Append(text[i]);
            }

            Console.WriteLine(sb);
        }
    }
}
